src/lib/urlManager.js

```diff
@@ -0,0 +1,57 @@
+import qs from 'qs';
```

Comment:
Is `qs` any better/different than using Node's [**querystring**](https://nodejs.org/api/querystring.html) module?

Reply:
Huh... I didn't know this was built into Node. That should work. I'll take a look. Thank you! :smiley:

Reply:
*querystring* works great. Great tip. Thank you!